napalm/base/__init__.py

```diff
@@ -87,12 +87,14 @@ def get_network_driver(module_name, prepend=True):
         try:
             module = importlib.import_module(module_name)
             break
-        except ImportError:
-            pass
+        except ModuleNotFoundError as e:
+            if e.name == "custom_napalm" or e.name in module_name:
+                continue
+            raise e
     else:
         raise ModuleImportError(
             'Cannot import "{install_name}". Is the library installed?'.format(
-                install_name=module_install_name
+                install_name=name
```

Comment:
Should we do something more here and print out the three different names we tried to import (in some way)?